tools/Environments/DevHome.Environments/ViewModels/LandingPageViewModel.cs

```diff
@@ -18,6 +18,7 @@
 using DevHome.Common.Services;

```

Comment:
All the changes on this page are part of the "Grouping" revert, except for small changes to the SearchHandler and adding enum to SortHandler.